dev/archery/archery/crossbow/reports.py

```diff
@@ -160,6 +215,29 @@ def url(self, query):
         repo_url = self.job.queue.remote_url.strip('.git')
         return '{}/branches/all?query={}'.format(repo_url, query)
 
+    def todayStr(self):
```

Comment:
I think we can now remove those three `todayStr`, `tasksToDict` and `getJsonTasks` functions given that we are moving them to JSONReporter

Reply:
Agreed.

Reply:
done... with fear
there was a "todo remove print" in getJsonTasks, which is copied in the json report